src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -25,6 +25,16 @@ class DateTimeNormalizer implements NormalizerInterface, DenormalizerInterface,
     const FORMAT_KEY = 'datetime_format';
     const TIMEZONE_KEY = 'datetime_timezone';
 
+    /**
+     * While denormalizing, we try to cast strings into real DateTime instances
+     * causing exception to be thrown if the string is not well formatted.
+     *
+     * Setting this tag to true will allow you to retrieve real instance of DateTime
+     * if the string is well formatted but will avoid exception if the string is not
+     * well formatted.
+     */
+    public const DATETIME_ALLOW_INVALID = 'datetime_allow_invalid';
```

Comment:
```suggestion
    public const ALLOW_INVALID = 'datetime_allow_invalid';
```

For consistency with the options defined above.

Reply:
I must agree that having a DATETIME_ALLOW_INVALID setted in the DateTimeNormalizer is a bit redundant :-)

Reply:
Change has been made right away ! ⚔️